src/web/routes/application/steps/decision/get-decision-status.test.js

```diff
@@ -60,7 +61,22 @@ const ELIGIBILITY_CONFIRMED_RESULT = {
   pregnantChildDOBMatch: 'not_set',
   qualifyingBenefits: 'not_set',
   identityOutcome: 'matched',
-  eligibilityOutcome: 'confirmed'
+  eligibilityOutcome: 'confirmed',
+  isPregnantOrAtLeast1ChildMatched: 'true'
+
+}
+
+const NOT_PREGNANT_AND_NO_CHILDREN_MATCHED_RESULT = {
```

Comment:
This doesn't seem like a very realistic VerificationResult, are we specifically setting all the verification outcomes to `NOT_SET` when the new flag is set? If not, then we should have some verification outcomes set for make this test be more accurate.

Reply:
I've updated the response to be matched for address line 1 and postcode and set the qualifying benefit to universal credit.

Reply:
`not_set` is not a value we're likely to get for our verification outcomes. `not_held` is much more likely for email & phone